backend/functions/offices/offices.function.ts

```diff
@@ -8,11 +11,13 @@ const httpTrigger: AzureFunction = async function (
   functionContext: Context,
   request: HttpRequest,
 ): Promise<void> {
-  const applicationContext = await applicationContextCreator(functionContext);
+  const applicationContext = await applicationContextCreator(functionContext, request);
   const officesController = new OfficesController(applicationContext);
 
   // get the offices from region 2 and return
```

Comment:
We should remove this old comment or turn it into a `TODO` about where to handle offices once we expand from region 2.